static/js/entry/root.js

```diff
@@ -7,15 +8,28 @@ import { AppContainer } from "react-hot-loader"
 import configureStore from "../store/configureStore"
 import Router, { routes } from "../Router"
 
+// Zendesk react module
+import Zendesk from "react-zendesk"
+
+const ZENDESK_KEY = SETTINGS.zendesk_config.help_widget_key
+const ZENDESK_ENABLED = SETTINGS.zendesk_config.help_widget_enabled
+
 const store = configureStore()
 
 const rootEl = document.getElementById("container")
 if (!rootEl) {
   throw new Error("Unable to find 'container' element")
 }
 
+const loadZendesk = () => {
+  return <Zendesk zendeskKey={ZENDESK_KEY} />
+}
+
 const renderApp = Component => {
   const history = createBrowserHistory()
+  if (ZENDESK_ENABLED && ZENDESK_KEY) {
+    loadZendesk()
```

Comment:
Are we not just adding zendesk into the footer of bootcamp project ? if yes then why these stuff is the part of this story ? as I remember, It was added to suggest relevant search articles of products.

Reply:
peter has already mentioned it in the ticket to add the search part for it.